engine/runtime-compiler/src/main/scala/org/enso/compiler/Passes.scala

```diff
@@ -97,15 +94,19 @@ class Passes(
       AliasAnalysis,
       DataflowAnalysis,
       CachePreferenceAnalysis,
-      // TODO passes below this line could be separated into a separate group, but it's more complicated - see usages of `functionBodyPasses`
```

Comment:
What about other linting passes, inside `org.enso.compiler.pass.lint` package, like `ModuleNameConflicts` and `ShadowedPatternFields`? Seems like we can remove those from interactive use as well.

Reply:
As I noted above - I think the only lint pass we should remove is Unused Bindings. Other passes can actually help in interactive mode - they can provide better error descriptions for issues that may be hard to debug otherwise.

Reply:
This is a valid statement. But unless you are manipulating code in the editor, IDE must not generate invalid programs that would trip on such passes.